lib/dash/dash_parser.js

```diff
@@ -766,6 +766,12 @@ shaka.dash.DashParser = class {
 
     context.adaptationSet = this.createFrame_(elem, context.period, null);
 
+    const isImage = context.adaptationSet.contentType == ContentType.IMAGE;
```

Comment:
If this is a temporary change, you should have a comment here saying that this should be removed later.

```js
// Filter image adaptation sets, until we add thumbnail track support.
// See: https://github.com/google/shaka-player/issues/559
```